src/pururu/application/events/event_system.py

```diff
@@ -33,6 +34,10 @@ def notify_listeners(self, data) -> None:
 class EventSystem:
     def __init__(self):
         self.events = {}
+        self.lastEmitted = None
```

Comment:
use `snake_case` for naming, docs: https://peps.python.org/pep-0008/#function-and-variable-names